Monika After Story/game/script-holidays.rpy

```diff
@@ -5562,20 +5562,20 @@ init 5 python:
 label mas_f14_monika_vday_origins:
     m 3eua "You'd like to learn about the history of Valentine's Day, [player]?"
     m 1rksdlc "It's quite dark, actually."
-    m 1euc "Its origin dates to as early as the second and third century in Rome, where Christianity had just been declared the official state religion."
-    m 3eud "Around this same time, a man known as Saint Valentine decided to go against the orders of Emperor Claudius II."
-    m 3rsc "Marriage had been banned because it was assumed that married men made poor soldiers."
-    m 3esc "Saint Valentine decided this was unfair and helped arrange marriages in secret."
-    m 1dsd "Unfortunately, he was caught and promptly sentenced to death."
-    m 1euc "However, while in custody, Saint Valentine fell in love with the jailer's daughter."
-    m 3euc "Before his death, he sent a love letter to her signed with 'From your Valentine.'"
-    m 1dsc "He was executed on February 14, 269 AD."
-    m 3eua "Such a noble cause, don't you think?"
+    m 1euc "The legends vary, but it dates back to the third century in Rome, when Christians were still persecuted by the Roman government."
+    m 3eud "Around this time, Emperor Claudius II had forbidden Christians from marrying, which a clergyman named Valentine decided was unfair."
+    m 3rsc "Against the orders of the emperor, he married Christians in secret."
+    m 3esc "Another version of the story is that Roman soldiers weren't allowed to be married, so by marrying people Valentine was saving them from conscription into the army."
+    m 1dsd "Either way, Valentine was caught and sentenced to death."
+    m 1euc "While in jail, he befriended the jailer's daughter and cured her blindness. Some say he even fell in love with her."
+    m 3euc "Unfortunately, this wasn't enough to save him. But before he died, he sent a letter to her, which he signed, "Your Valentine.""
```

Comment:
Consistency, should use single quotes within these. Additionally, double quotes would need to be escaped (`\"`") to work.
```suggestion
    m 3euc "Unfortunately, this wasn't enough to save him. But before he died, he sent a letter to her, which he signed, 'Your Valentine.'"
```